core/modules/replica/FindRequest.h

```diff
@@ -0,0 +1,196 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_REPLICA_FIND_REQUEST_H
+#define LSST_QSERV_REPLICA_FIND_REQUEST_H
```

Comment:
I think there should not be an underscore in FIND_REQUEST. It's in many files, if you're going to address it please check them all, I won't add noise by flagging any more.

Reply:
What you're suggesting ages against 4.9 of the LSST C++ Style Guide:
https://developer.lsst.io/cpp/style.html#header-files-must-include-a-construction-that-prevents-multiple-inclusion

Note, that the guide suggest to introduce '_' to separate words in the Camel-cased file names

Reply:
Style guide is wrong, or actually example there is wrong because there is no explicit suggestion to add underscores. Suppose you have two files in your package  `FindRequest.h` and `find_request.h`.

Reply:
You've convinced me. And the Style guide needs to be fixed (CC to @ktlim)